APISpecification.yaml

```diff
@@ -111,6 +111,55 @@ paths:
         404:
           description: Genome not found
           content: {}
+  /api/metadata/genome/{genome_id}/ftplinks:
+    get:
+      summary: Returns a list of FTP download links for the specified genome
+      parameters:
+      - name: genome_id
+        in: path
+        required: true
+        schema:
+          type: string
+          example: 3704ceb1-948d-11ec-a39d-005056b38ce3
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/FTPLink'
+        404:
+          description: Genome not found
+          content: {}
+  /api/metadata/genome/{genome_id}/checksum/{region_id}:
+    get:
+      summary: Returns a checksum for the specified top-level region
+      parameters:
+        - name: genome_id
+          in: path
+          required: true
+          schema:
+            type: string
+            example: 3704ceb1-948d-11ec-a39d-005056b38ce3
+        - name: region_id
+          in: path
+          required: true
+          schema:
+            type: string
+            example: "13"
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: string
+                example: 428d56c3fc01ad3a8db49830fb587625
```

Comment:
What's the successful response looks like ? Specification says response is of `application/json` but  `428d56c3fc01ad3a8db49830fb587625` is not valid json.

Reply:
Fixed